neqo-crypto/src/aead_fuzzing.rs

```diff
@@ -7,29 +7,51 @@
 use crate::constants::{Cipher, Version};
 use crate::err::{sec::SEC_ERROR_BAD_DATA, Error, Res};
 use crate::p11::SymKey;
+use crate::RealAead;
 use std::fmt;
 
 pub const FIXED_TAG_FUZZING: &[u8] = &[0x0a; 16];
-pub struct Aead {}
 
-#[allow(clippy::unused_self)]
-impl Aead {
-    pub fn new(_version: Version, _cipher: Cipher, _secret: &SymKey, _prefix: &str) -> Res<Self> {
-        Ok(Self {})
+pub struct FuzzingAead {
+    real: Option<RealAead>,
+}
+
+impl FuzzingAead {
+    pub fn new(
+        fuzzing: bool,
+        version: Version,
+        cipher: Cipher,
+        secret: &SymKey,
+        prefix: &str,
+    ) -> Res<Self> {
+        let real = if fuzzing {
+            None
+        } else {
+            Some(RealAead::new(false, version, cipher, secret, prefix)?)
+        };
+        Ok(Self { real })
     }
 
     #[must_use]
     pub fn expansion(&self) -> usize {
+        if let Some(aead) = &self.real {
+            return aead.expansion()
+        }
+
         FIXED_TAG_FUZZING.len()
```

Comment:
```suggestion
            aead.expansion()
        } else {
            FIXED_TAG_FUZZING.len()
        }
```

Courtesy of the formatter (which complained about the lack of semicolon).